### ISSUE TITLE
Mydomain 

### ISSUE THREAD

**Issue by jmartins1** [snipe/snipe-it#7402]
Question:

I host SnipeIt internally on a WAMP server. I created a DNS record to navigate to SnipeIT by name. After adding an additional DHCP scope, I am no longer able to navigate to SnipeIT by name, but can by IP address. Attempting to use the previous DNS entry redirects me to a page snipeit.mydomain.com. Is like to know where this page is being generated from and how it can be corrected in the app side. For now I've created a C record, which corrected the issue but obviously this should need to be done. Ideas, suggestions? 

Thanks in advance.

**Comment by ACurnow** [comment]
@jmartins1 Have you replaced the value in the APP_URL setting in your .env file?

**Comment by jmartins1** [comment]
My App_URL points directly to an internal server address. By entering the server address the DNS redirects the request to the WAMP server to server up SnipeIT. After I added an additional DHCP scope it stopped working; it would direct me to a snipeit.mydomain.com page. I had to add a C record to correct this. I’d like to know where the “mydomain.com” is being served from. I believe this is a SnipeIT setting.  

 

 

 

Jose Martins, M.Ed., MCSA, MCP

IT Administrator
